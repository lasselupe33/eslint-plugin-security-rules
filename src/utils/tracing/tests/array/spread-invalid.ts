/**
 * In case we cannot properly determine the value of the given index then the
 * whole array should be traced
 */

const arrA = ["aVal", "bVal"];
// @ts-expect-error Purposely invalid code
const arrB = [...arrA, "cVal", ...invalid, ...["eVal", "fVal"]];

const start = arrB[4];

export {};
