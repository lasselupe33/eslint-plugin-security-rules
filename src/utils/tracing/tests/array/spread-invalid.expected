aValbValcVal__Unable to resolve identifier to variable__eValfVal